provider/virtio_rng/check_points.py

```
@@ -0,0 +1,68 @@
+"""Helper functions for virtio device testing"""
+
+import aexpect
+import logging as log
+
+from avocado.utils import process
+
+logging = log.getLogger('avocado.' + __name__)
+
+def check_guest_dump(session, exists=True):
```

If your intention is primarily to check functions or state of  system , normally we don't expect it will return one value. 
Return one value means your method begin with get_xx.

---

If this is a checkpoint, it is ok to directly raise an exception when checking fails instead of returning an error message and check the error message outside, then raise it. What do you think of?

---

The function now contains only exceptions and returns nothing.